fabfile/adage_server.py

```diff
@@ -168,11 +158,19 @@ def rebuild_search_index():
 @task
 def import_data_and_index():
     """
-    invoke import_data, which manually links to the get_pseudo_sdrf.py
-    file extracted from the get_pseudomonas repository
+    invoke import_data (which manually links to the get_pseudo_sdrf.py
+    file extracted from the get_pseudomonas repository) and
+    import_activity with sample data from CONFIG['data']
     """
-    run('python manage.py import_data "%s"' % CONFIG['data']['annotation_file'])
+    run('python manage.py import_data "%s"' %
+        CONFIG['data']['annotation_file'])
     rebuild_search_index()
+    run('python manage.py organisms_create_or_update --taxonomy_id=208964 '
```

Comment:
Yes! Just haven't gotten to that yet. We need sample data files under `data/` or a script step that downloads a file, and I need to know how to run those commands. Maybe we should track this an an issue?